scripts/execution_layer.py

```diff
@@ -146,6 +169,41 @@ def do_generate_lib(args):
             generate_lib(lib)
 
 
+def do_merge(args):
+    from_module = impl(args.feature)
+    if not from_module.is_file():
+        raise Exception(f"'{args.feature}' does not exist.")
+
+    to_module = impl(args.base)
+    if not to_module.is_file():
+        raise Exception(f"'{args.base}' does not exist.")
+
+    print("Calculating change...", file=stderr)
+    changes = patch(args.feature) or ""
```

Comment:
Do we want to continue doing work if `changes` ends up being `""` here?

Reply:
Yeah, good point, we can stop at that point -- this is copypasta from `rebase`.